src/modules/stake-dot/screens/StakeDot/StakeDot.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useCallback } from 'react';
+import { useDispatch } from 'react-redux';
+import { StakeDotAlert } from '../../components/StakeDotAlert';
+import { useInitEffect } from '../../../../common/hooks/useInitEffect';
+import { Dashboard } from '../../components/Dashboard';
+import { AvalancheActions } from '../../../stake-avax/actions/AvalancheActions';
+
+const needAlert = false;
+
+export const StakeDot = () => {
+  const dispatch = useDispatch();
+
+  useInitEffect(() => {
+    dispatch(AvalancheActions.fetchTransactionStatus());
+  });
+
+  const renderAlert = useCallback(() => <StakeDotAlert />, []);
+
+  if (needAlert) {
+    // TODO: check connection and requiredNetwork
+    return renderAlert();
+  }
```

Comment:
I don't understand why we have to use an additional callback to render the alert?

```suggestion
  if (needAlert) {
    // TODO: check connection and requiredNetwork
    return <StakeDotAlert />;
  }
```